dowhy/causal_estimators/causalml.py

```diff
@@ -0,0 +1,108 @@
+        
+        # Add the observed confounders and one hot encode the categorical variables
+        self._observed_common_causes_names = self._target_estimand.backdoor_variables
+        if self._observed_common_causes_names:
+            # Get the data of the unobserved confounders
+            self._observed_common_causes = self._data[self._observed_common_causes_names]
+            # One hot encode the data if they are cateogorical
+            self._observed_common_causes = pd.get_dummies(self._observed_common_causes, drop_first=True)
+        else:
+            self._observed_common_causes = []
+        
+        # Check the instrumental variables involved
+        self.logger.debug("Instrumental variables used:"+
+                        ",".join(self._target_estimand.instrumental_variables))
+        
+        # Perform the same actions as the above
+        self._instrumental_variable_names = self._target_estimand.instrumental_variables
+        if self._instrumental_variable_names:
+            self._instrumental_variables = self._data[self._instrumental_variable_names]
+            self._instrumental_variables = pd.get_dummies(self._instrumental_variables, drop_first=True)
+        else:
+            self._instrumental_variables = []
+
+        #Check if effect modifiers are used
+        self.logger.debug("Effect Modifiers used:" + 
+                        ",".join(self._effect_modifier_names))
+        
+        
+        # Get the class corresponding the the estimator to be used
+        estimator_class = self._get_causalml_class_object(self._causalml_methodname)
+        # Initialize the object
+        # pdb.set_trace()
+        self.estimator = estimator_class(**self.method_params["init_params"])
+        self.logger.info("INFO: Using CausalML Estimator")
+        self.symbolic_estimator = self.construct_symbolic_estimator(self._target_estimand)
+        self.logger.info(self.symbolic_estimator)
+        
+    def _get_causalml_class_object(self, module_method_name, *args, **kwargs):
+        
+        try:
+            (module_name, _, class_name) = module_method_name.rpartition(".")
+            estimator_module = import_module(module_name)
+            estimator_class = getattr(estimator_module, class_name)
+
+        except (AttributeError, AssertionError, ImportError):
+            raise ImportError('Error loading {}.{}. Double-check the method name and ensure that all econml dependencies are installed.'.format(module_name, class_name))
+        return estimator_class
+
+    def _estimate_effect(self):
+        X_names = self._instrumental_variable_names + \
+                self._observed_common_causes_names + \
+                self._effect_modifier_names
+
+        y_name = self._outcome_name # We need to make sure that both are pd.Series, so we take the first element
+        treatment_name = self._treatment_name[0] # As we have only one treatment variable

```

Comment:
Does causalml allow multiple treatment variables? If so,  you can consider fixing this for a future pull request.

Reply:
![image](https://user-images.githubusercontent.com/17275495/78328562-158b7e00-759d-11ea-9850-cf1424acbe58.png)

I think it only takes a 1D array and thus, I had to explicitly pass the element in the treatment name as 'v0' and not ['v0'] to prevent a shape mismatch